JiraPS/Public/Set-JiraIssueLabel.ps1

```diff
@@ -1,4 +1,4 @@
-﻿function Set-JiraIssueLabel {
+function Set-JiraIssueLabel {
```

Comment:
this change is switching from UTF8-BOM to UTF8, right?

interesting... as I will change all of the files to UTF8-BOM 😆

Reply:
I started to use VS Code for PowerShell.
Therefore, I lose a control of encoding.

Just to confirm: Should I convert all files to UTF8-BOM?

Reply:
As you prefer. You can leave it as-is.

VS Code allows you to change the encoding of a file by clicking here:
![image](https://user-images.githubusercontent.com/4992870/64955769-664dba80-d889-11e9-863a-297f81b21a95.png)
or with "Change File Encoding" when pressing `Ctrl+Shift+P`

Reply:
Thank you!